gpt-survey-summarizer/scripts/database-dump-export.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+set -e
+
+DATE="$(date +%Y_%m_%d_%H_%M)"
+FILENAME="$MINA_PLATFORM-gpt-survey-summarizer-dump-$DATE.rdb"
+
+echo "INFO: Installing tools"
+apk add aws-cli >/dev/null;
+
+echo "INFO: Exporting dump of database"
+redis-cli -h gpt-survey-summarizer-redis-master --pass $REDIS_PASSWORD --rdb $FILENAME
```

Comment:
**suggestion:** I think we should use a variable to point to the server, and avoid to hardcode it in the script. At the moment, that would only work when the helm release is named `gpt-survey-summarizer`.